_guides/responsive_amp.md

```diff
@@ -5,17 +5,123 @@ order: 0
 folder: responsive
 ---
 
-AMP balances its support for developers and users.
-It helps developers create content that looks and behaves as designed,
-while protecting the user experience from slow page loads
-and juttery interactions.
-
-In AMP, only inline styles are allowed; this removes extra HTTP requests
-from the critical rendering path.
-AMP uncouples document layout from resource loading;
-it requires all external resources to state their size
+Create responsive pages in AMP following these style and layout principles.
```

Comment:
I think we should start with.

"It is super easy to make responsive elements in AMP. Just put `layout=responsive` on them".

We definitely should also have a concrete example up top for images, because they are 10x more important than everything else. And if possible provide screenshots showing the effect of `layout=responsive`